hackathon_site/dashboard/frontend/src/slices/users/userSlice.js

```diff
@@ -92,11 +100,52 @@ export const logIn = createAsyncThunk(
     }
 );
 
+// export const acknowledge = createAsyncThunk(
+//     `${userReducerName}/acknowledge`,
+//     async({signature}, {dispatch, rejectWithValue}) => {
+//         try {
+//             const response = await post("/api/acknowledge",{signature});
```

Comment:
This will eventually correspond to our Profile model: https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/event/models.py#L27

Sending an API request will set two values in the model: `acknowledge_rules` and `e_signature`. The easiest way for us to do this is just to make a PATCH (i.e. change) endpoint for the profile model, which takes in the fields you want to change. So this would probably be something like:

```javascript
const response = await patch("/api/event/profile", {e_signature: signature, acknowledge_rules: true})
```

`patch()` currently doesn't exist yet either, so that would need to be written too as a separate PR.